ansible/vars/topo_tgen-t1-64-3.yml

```diff
@@ -0,0 +1,159 @@
+topology:
+  disabled_host_interfaces:
+    - 0
+    - 1
+    - 2
+    - 3
+    - 4
+    - 5
+    - 6
+    - 7
+    - 8
+    - 9
+    - 10
+    - 11
+    - 12
+    - 13
+    - 14
+    - 15
+    - 16
+    - 17
+    - 18
+    - 19
+    - 20
+    - 21
+    - 22
+    - 23
+    - 24
+    - 25
+    - 26
+    - 27
+    - 28
+    - 29
+    - 30
+    - 31
+    - 32
+    - 33
+    - 34
+    - 35
+    - 36
+    - 37
+    - 38
+    - 39
+    - 40
+    - 41
+    - 42
+    - 43
+    - 44
+    - 48
+    - 49
+    - 50
+    - 51
+    - 52
+    - 53
+    - 54
+    - 55
+    - 56
+    - 57
+    - 58
+    - 59
+    - 60
+    - 61
+    - 62
+    - 63
+
+  VMs:
+    ARISTA01T2:
+      vlans:
+        - 45
+      vm_offset: 0
+    ARISTA01T0:
+      vlans:
+        - 46
+      vm_offset: 1
+    ARISTA02T0:
+      vlans:
+        - 47
+      vm_offset: 2
+
+configuration_properties:
+  common:
+    dut_asn: 64607
+    dut_type: LeafRouter
+    nhipv4: 10.10.246.254
+    nhipv6: FC0A::FF
+    podset_number: 200
+    tor_number: 16
+    tor_subnet_number: 2
+    max_tor_subnet_number: 16
+    tor_subnet_size: 128
+  spine:
+    swrole: spine
+  tor:
+    swrole: tor
+
+configuration:
+  ARISTA01T2:
+    properties:
+    - common
+    - spine
+    bgp:
+      asn: 65200
+      peers:
+        64607:
+        - 1.0.0.2
+        - FC00::2
+    interfaces:
+      Loopback0:
+        ipv4: 100.1.0.3/32
+        ipv6: 2064:100::3/128
+      Ethernet1:
```

Comment:
For the t1 topo, we would like to have at least 1 PortChannel interface. The other 1 can be regular interfaces

Reply:
These are ixia devices - so the actual topo config does not matter. This is more for the sake of completeness.

Reply:
We need to ensure that there are no issues when RDMA traffic enters/leaves portchannel interface/ pfc frames entering/exiting via member ports of a portchannel. Hence the need to simulate this

Reply:
Done